salt/utils/network.py

```diff
@@ -1006,9 +1006,6 @@ def win_interfaces():
     '''
     Obtain interface information for Windows systems
     '''
-    if WIN_NETWORK_LOADED is False:
```

Comment:
If you do remove those, do you still need the WIN_NETWORK_LOADED at all?

Reply:
Yesh, I removed WIN_NETWORK_LOADED